language-server/modules/langserver-core/src/test/resources/completion/new/type_def/config/config1.json

```diff
@@ -403,4 +411,4 @@
       "insertTextFormat": "Snippet"
     }
   ]
```

Comment:
Broken test identifies the following incorrectly

```
import ballerina/module1;

public type TEST_TYPE

function testFunction1() <cursor>

function testFunction() {
}
```

Earlier it identified the cursor as function def and now it is module variable def

Reply:
With the new change `function`(without signature) takes as type descriptor. So I think this recovery is correct. Isn't it?
```
import ballerina/module1;
public type TEST_TYPE function  MISSING[;] 
MISSING[@]testFunction1()  MISSING[] MISSING[=]<cursor>function  INVALID[testFunction]() {} MISSING[;]
```

Reply:
Seem like recovery is correct.

Reply:
Please disable this particular test case only

Reply:
Disabled. Create a separate issue(https://github.com/ballerina-platform/ballerina-lang/issues/28990) to track this.